actions/views/mfa.js

```diff
@@ -29,6 +29,9 @@ export function generateMfaSecret() {
     };
 }
 
+/**
+ * @deprecated Clients should attempt to login without MFA and check if MFA token error is returned to show MFA prompt
```

Comment:
Do we use `@deprecated` elsewhere? I'm not sure if we have a consistent way of marking deprecated methods in comments.

Alternatively, this is just part of the web app, so we should be able to completely remove this function if we're not using it any more

Reply:
Removing it completely sounds good to me.